commands/host.vegman

```diff
@@ -158,25 +158,66 @@ function cmd_virtualmedia {
   subcommand "$@"
 }
 
+# @sudo cmd_virtualmedia_status admin
+# @restrict cmd_virtualmedia_status admin
+# @doc cmd_virtualmedia_status
+# Get status of the virtual media
+function cmd_virtualmedia_status {
+  local usb_status_metadata=( "USB0;Proxy" "USB1;Legacy" )
+
+  for usb_item_metadata in "${usb_status_metadata[@]}"
+  do
+    local id="${usb_item_metadata%%;*}"
+    local vm_type="${usb_item_metadata#*;}"
```

Comment:
why `id` was greedy (`%%`), but this one isn't (`#` instead of `##`) ?
Doesn't make any actual difference in any of the cases, but still.

Reply:
Yes, in this case, it doesn't matter. I guess this is just a mistake. To uniform, I'll add `##` to grep `vm_type`.

Reply:
fixed